src/api/app/services/scm_status_reporter.rb

```diff
@@ -1,4 +1,7 @@
 class SCMStatusReporter
+  REPORT_FREE_PULL_REQUEST_ACTIONS = %w[closed].freeze
+  REPORT_FREE_MERGE_REQUEST_ACTIONS = %w[close merge].freeze
```

Comment:
What about abstracting it not only to PR/MR with just one collection of events to skip?

```suggestion
  EVENT_ACTIONS_TO_SKIP = %w[closed close merge].freeze
```